src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -77,14 +77,14 @@
         public static final int FLYWHEEL_1 = 0, FLYWHEEL_2 = 0, TURRET = 0, HOOD = 0;
 
         // intake can ids are range 30-39
-        public static final int INTAKE_1 = 0, INTAKE_2 = 0, INTAKE_UP = 0, INTAKE_DOWN = 0;
+        public static final int INTAKE_1 = 0, INTAKE_2 = 0, INTAKE_UP = 1, INTAKE_DOWN = 0;
 
         // index can ids are range 40-49
         public static final int INDEX_INGEST_MOTOR = 40, INDEX_FEEDER_MOTOR = 41, INDEX_INGEST_SENSOR = 4,
                 INDEX_FEEDER_SENSOR = 5;
 
         // climb can ids are range 50-59
-        public static final int CLIMB_DYNAMIC = 0, CLIMB_FIXED = 0, CLIMB_ANGLE_UP = 0, CLIMB_ANGLE_DOWN = 0;
+        public static final int CLIMB_DYNAMIC = 50, CLIMB_FIXED = 51, CLIMB_ANGLE_UP = 52, CLIMB_ANGLE_DOWN = 53;
 
```

Comment:
Angle_up and down are solenoids, which are indexed differently since they are pneumatic port #s. Please update the constant names to differentiate motor CAN IDs from pneumatic port IDs and choose valid pneumatic port #s.